nix/workbench/shell.nix

```diff
@@ -87,7 +85,6 @@ in project.shellFor {
   # These programs will be available inside the nix-shell.
   nativeBuildInputs = with pkgs; with haskellPackages; with cardanoNodePackages; [
     cardano-ping
-    cabalWrapped
```

Comment:
Ok, this, I think, is the biggest problem I can see in this PR -- this very specifically breaks Cabal-inside-Nix-shell.

I think we agreed in the past we shouldn't break it.

Reply:
Can you be specific about what you think is broken? I think it all works.

Reply:
cabalWrapped fixes cabal's behaviour of always recompiling source-repository-packages, this does not happen anymore since dependencies are now (mostly) from a repository.

Reply:
@michaelpj, well:

```

Reply:
The `cardano-node` `master` branch has this bit of functionality which explains the basic workbench workflow: https://github.com/input-output-hk/cardano-node#workbench-a-local-cluster-playground

Reply:
Oh I see, we just need to provide `cabal` instead. I just assumed it was there!

Reply:
If both `cabal` and `cabalWrapped` were there, there'd be a clash, because both of them provide the same executable name.

Reply:
@michaelpj, but a question -- what is the motivation for `cabalWrapped` removal, in the first place?

Can this PR be made to work with `cabalWrapped` still there?  How difficult would that be?

Reply:
It's not needed, because `cabal` just works - like it will for every other project.

Reply:
ok, we can fix this, thanks for letting us know

Reply:
I know you have a slightly different workflow, Serge, I tried to discuss that a bit here: https://docs.google.com/document/d/1PGmrvcNmfLP7HhGZpcgEee6zlHsQ-hozCC2XkjzHcbI/edit#heading=h.q6n5jsdrrm3b

If you find you still use `source-repository-package` lots then `cabalWrapped` might still be useful for you.

Reply:
@michaelpj, running node clusters with feature branches is something not specific to the performance team.

I've made a reply in the document: https://docs.google.com/document/d/1PGmrvcNmfLP7HhGZpcgEee6zlHsQ-hozCC2XkjzHcbI/edit?disco=AAAAf9tvVhs

In fact, we specifically want to encourage such use -- precisely to enable people to get direct, early feedback on their code changes.

Reply:
In short, I don't think we have evidence that `cabalWrapped` is necessary in this (otherwise great!) PR.

I propose we undo that change and reinstate it.

Reply:
I put `cabalWrapped` back. Thanks @deepfire.